backend/services/src/activity/activity.service.ts

```diff
@@ -986,6 +987,16 @@ export class ActivityService {
 			);
 		}
 
+		if (user.validatePermission===ValidateEntity.CANNOT) {
```

Comment:
Shouldn't this be inside the validate activity function?

Reply:
Fixed.